### PR TITLE
chore: Migrate /view_model/content to camelCase

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/page/template/content/preferences-account.htm [hunk 1]
```diff
@@ -24,58 +24,58 @@
...
+          <div class="preferences-account-mail" data-bind="text: selfUser().email()" data-uie-name="enter-email"></div>
+          <div class="preferences-account-phone" data-bind="text: selfUser().phone()" data-uie-name="enter-phone"></div>
+          <accent-color-picker class="preferences-account-accent-color preferences-section-account-space-before" params="user: selfUser(), selected: changeAccentColor" data-uie-name="enter-accent-color"></accent-color-picker>
         </div>
 
         <label class="preferences-account-picture-button" for="self-upload-file-input" data-bind="l10n_tooltip: z.string.tooltipPreferencesPicture">
-          <participant-avatar params="participant: self_user, size: z.components.ParticipantAvatar.SIZE.X_LARGE"></participant-avatar>
+          <participant-avatar params="participant: selfUser, size: z.components.ParticipantAvatar.SIZE.X_LARGE"></participant-avatar>
           <input id="self-upload-file-input"
                  type="file" accept="image/jpg,image/jpeg,image/png,image/bmp"
```
Comment (AndyLnd) [inline app/page/template/content/preferences-account.htm]: the ```accept``` here is obsolete

File: app/script/view_model/content/PreferencesAVViewModel.js [hunk 2]
```diff
@@ -76,48 +88,51 @@ z.viewModel.content.PreferencesAVViewModel = class PreferencesAVViewModel {
...
-        this.available_devices.audio_input().length,
-        this.available_devices.video_input().length
-      )
-      .then(({media_stream_constraints, media_type}) => {
-        return this.media_stream_handler.request_media_stream(media_type, media_stream_constraints);
+    const requestAudio = !!this.availableDevices.audio_input().length;
+    const requestVideo = !!this.availableDevices.video_input().length;
+    return this.constraintsHandler
+      .get_media_stream_constraints(requestAudio, requestVideo)
+      .then(({media_stream_constraints, mediaType}) => {
```
Comment (Yserz) [inline app/script/view_model/content/PreferencesAVViewModel.js]: streamConstraints

File: app/script/entity/User.js [hunk 3]
```diff
@@ -159,9 +159,9 @@ z.entity.User = class User {
     this.isTemporaryGuest = ko.observable(false);
     this.is_team_member = ko.observable(false);
     this.team_role = ko.observable(z.team.TeamRole.ROLE.NONE);
-    this.is_team_manager = ko.pureComputed(() =>
-      [z.team.TeamRole.ROLE.ADMIN, z.team.TeamRole.ROLE.OWNER].includes(this.team_role())
-    );
+    this.isTeamManager = ko.pureComputed(() => {
+      return [z.team.TeamRole.ROLE.ADMIN, z.team.TeamRole.ROLE.OWNER].includes(this.team_role());
```
Comment (Yserz) [inline app/script/entity/User.js:163]: could be a one-liner

Comment (gregor) [reply app/script/entity/User.js:163]: Does not fit

**Pull request merged**